public/locales/en/common.json

```diff
@@ -84,7 +84,8 @@
     "NEW_FACILITATOR": "New Facilitator",
     "SEARCHBAR_PLACEHOLDER": "Search Facilitators..",
     "FACILITATOR_CREATED_SUCCESSFULLY": "Teacher has been Successfully Created",
-    "FACILITATOR_UPDATED_SUCCESSFULLY": "Teacher has been Successfully Updated!"
+    "FACILITATOR_UPDATED_SUCCESSFULLY": "Teacher has been Successfully Updated!",
```

Comment:
@AkshataKatwal16 keep first letter of the sentence as capital rest in small case

Reply:
updated